lib/galaxy/tools/actions/upload_common.py

```diff
@@ -377,7 +377,7 @@ def create_paramfile(trans, uploaded_datasets):
             # TODO: This will have to change when we start bundling inputs.
             # Also, in_place above causes the file to be left behind since the
             # user cannot remove it unless the parent directory is writable.
-            if link_data_only == 'copy_files' and trans.user:
+            if link_data_only == 'copy_files' and trans.user and trans.app.config.external_chown_script:
```

Comment:
```suggestion
            if link_data_only == 'copy_files' and trans.user and trans.app.config.external_chown_script is not None:
```

Reply:
Why do you prefer checking for `None`? Is there a different meaning in an empty string?

Reply:
Ah, I see - my mistake. Actually you make this more consistent. 

Question is if we want to / should assume that this comes as `None` from the configuration or empty string .. or if we don't care. 

Anyway I'm fine with the change.